internal/message/message.go

```diff
@@ -7,11 +7,44 @@ import (
 	"time"
 )
 
+// ShuffleStringSlice is a string slice with counter and shuffle methods
+type ShuffleStringSlice struct {
+	s []string
+	i int
+}
+
+// UnmarshalYAML parses string array into a slice and initializes a counter
+func (s *ShuffleStringSlice) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var str []string
+	err := unmarshal(&str)
+	s.s = str
+	return err
+}
+
+// ShuffleWith does a shuffle based on a shuffle function
+func (s *ShuffleStringSlice) ShuffleWith(sf func([]string)) {
+	sf(s.s)
+}
+
+// GetNext returns a next string from a shuffled slice
+// it will begin from the first element if the counter will reach the end of the slice
+func (s *ShuffleStringSlice) GetNext() string {
+	if len(s.s) == 0 {
+		return ""
+	}
+
+	s.i++
+	if s.i == len(s.s) {
+		s.i = 0
+	}
```

Comment:
We need to reshuffle the array when meet the end, or we constantly would get the same sequence of phrases.

Reply:
Makes sense

Reply:
Done